spec/controllers/api/v2/accounts/report_controller_spec.rb

```diff
@@ -223,6 +223,39 @@
         expect(response).to have_http_status(:success)
       end
     end
+
+    context 'when an agent has access to multiple accounts' do
+      let(:account1) { create(:account) }
+      let(:account2) { create(:account) }
+
+      let(:params) do
+        super().merge(
```

Comment:
For my understanding, what does this do?

Reply:
there is a params object defined in the parent's block.. super will access that and then the additional params will be merged into it